bin/wpc_init

```diff
@@ -121,7 +134,22 @@ set -e
 ## This code will be run during setup, INSIDE the container.
 ##
 
-wp core install --skip-email --admin_user=admin --admin_password=admin --admin_email=admin@localhost.invalid --url=http://localhost --title=Site
-# wp theme activate ...
-# wp plugin activate ...
+##############
+# Config
+##############
+title=your_site_title
```

Comment:
To be more illustrative to the user, could we use something that looks more like a title here? Such as `title="Your Site Title"`. Assuming some users won't be very familiar with shell scripting, they may not be aware that they have to surround the thing with quotes if they want to add spaces.